src/components/IconButton/IconButton.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { JSXElementConstructor } from 'react';
+
+import styles from './IconButton.module.scss';
+import { IconProps } from '../Icons/Icons.types';
+
+export interface Properties {
+  className?: string;
+  onClick: () => void;
+
+  Icon: JSXElementConstructor<IconProps>;
+  label?: string;
+  size?: string | number;
+  isFilled?: boolean;
+}
+
+export class IconButton extends React.Component<Properties> {
```

Comment:
Can we write this as a function component to be consistent with the other components in zUI?

Reply:
Sure. This component is simple enough.